postgres/experiment.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: redskyops.dev/v1alpha1
 kind: Experiment
 metadata:
-  name: postgres-example
+  name: postgres-example-test-cost-metric
 spec:
   optimization:
   - name: "experimentBudget"
-    value: "40" #number of trials
+    value: "100" #number of trials
```

Comment:
this was set to 40 because we usually recommend 20*parameters

Reply:
Nice catch.